### DISCUSSION TITLE
Mesh Size is 0 although I can see it in Baram Flow

### DISCUSSION THREAD

**Discussion by parisipa** [nextfoam/baram#114] [Q&A]
Hello, everyone.

I have installed version 24.3.3. in Windows and was attempting to run a simulation in BARAM Flow. The mesh gets imported and I can see it on the UI. I am able to change all the settings and run the simulation, but the calculations ends as soon as it starts. I discovered that when I click on mesh info, the number of cells is interpreted as 0. Any ideas as to why that happens?

The result is that my simulation in terminated immediately when the calculation starts. 

I ran the exact same case on another computer and it was fine. Unfortunately I can't share the files due to confidentiality issues, but any feedback would be appreciated.

Thank you in advance.

![image](https://github.com/user-attachments/assets/e3e524d2-e0d9-4951-8814-02cd3e967f24)

**Comment by kdg5424** [comment]
Hello @parisipa. This is Donggyu from NEXTFOAM.
Is there any cell zone and multiple region in your mesh?

Overall, your mesh look weird. Because the Largest cell is 0 and the smallest cell is 1e+9.
So the mesh generation looks to have gone wrong.

Could you check your cell zone and mutiple region that they are all water-tight?

Best regards

Donggyu kim

**Comment by parisipa** [comment]
Hi, Donggyu

My geometry (single region) is okay. I can confirm, because I have managed to run it on a Baram version locally installed on my laptop. However, when I tried to perform the same calculation on a work station for more computational power, I receive this weight mesh size info. I tested it with other Baram cases I have created with different geometries and it gives the exact same numbers.

Any idea what would be different from one machine to the other? Since in both cases I tried the same Baram version, I don't believe it's a Baram issue.

Thanks in advance,
Angie

**Comment by kdg5424** [comment]
Hello @parisipa. This is Donggyu. Sorry for the late reply.
We should do some test. So could you let me know your case, model?
Just let me know simple things like turbulence model what you used, energy equation on/off, VOF or CHT and etc.
We'll do test the case as simillar as yours.

Best regards

Donggyu Kim

**Reply by parisipa** [reply]
Hi Donggyu. Let's try for example the mixing elbow case with  a k-omega SST model, no energy equation. To simulate my problem, try to access Baram installed locally on a machine that you are accessing via VPN and remote desktop. Maybe the issue lies somewhere there.

**Reply by kdg5424** [reply]
Hello @parisipa.
Are the both machine (local machine and remote desktop) windows os? Or are they different?
The problem sounds very simple not complicated. Although I haven't tested yet, but I also agree that there is no problem with BARAM and settings.
As soon as you let me know your os, I will test it.

Best regards

Donggyu Kim

**Reply by parisipa** [reply]
Yes, both machines are Windows OS. Thanks in advance for checking.
If another program is using the MPI protocol, could that be the source of the problem?
Kind regards,
Angie

**Reply by kdg5424** [reply]
Hello @parisipa.
We did test the two machines with Windows OS. And we couldn't find any problems.
So could you check your mpi setting as I told #116?

Best regards
Donggyu Kim
